src/lib/Microsoft.Health.Fhir.Ingest/Service/ResourceIdentityServiceAttribute.cs

```diff
@@ -0,0 +1,20 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Fhir.Ingest.Host
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = true, Inherited = false)]
+    public sealed class ResourceIdentityServiceAttribute : Attribute
+    {
+        public ResourceIdentityServiceAttribute(string type)
```

Comment:
You can have an alternative ctor that takes your enumeration. i.e. public ResourceIdentityServiceAttribute(ResourceIdentityServiceType type) : this(type.ToString())

Reply:
Thanks for this suggestion. Updated.